solidity/random-beacon/tasks/increase-authorization.ts

```diff
@@ -0,0 +1,75 @@
+import { task, types } from "hardhat/config"
```

Comment:
We should keep `initialize.ts` script as it has improvements from https://github.com/keep-network/keep-core/pull/3110.

Reply:
Those improvements were copied & pasted to each task that was broken down from `initialize.ts` script. Please see that it should be the same as in 3110. The reason why I removed that script is that in previous PRs we talked about having 4 separate tasks and having these tasks in separate files make them more flexible for reuse. Ex. in `random-beacon` we use: mint.ts, stake.ts, authorization.ts, and register.ts. In `ecdsa` we just copy authorization.ts, and register.ts, cause we don't need mint.ts and stake.ts there.

Reply:
Please see https://github.com/keep-network/keep-core/pull/3123 for tasks improvements and implementation for ECDSA.

Ideally, I'd keep the implementation of the task out of this PR, as it gets too complicated.